integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/mock/answer/DefaultResponseConfigure.java

```diff
@@ -0,0 +1,118 @@
+package com.sequenceiq.it.cloudbreak.dto.mock.answer;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.hamcrest.Matcher;
+
+import com.sequenceiq.it.cloudbreak.dto.CloudbreakTestDto;
+import com.sequenceiq.it.cloudbreak.dto.mock.Method;
+import com.sequenceiq.it.cloudbreak.mock.ExecuteQueryToMockInfrastructure;
+import com.sequenceiq.it.cloudbreak.testcase.mock.response.MockResponse;
+
+public class DefaultResponseConfigure<T extends CloudbreakTestDto> {
+    private final Method method;
+
+    private final String path;
+
+    private final T testDto;
+
+    private final ExecuteQueryToMockInfrastructure executeQuery;
+
+    private final Map<String, String> pathVariables = new HashMap<>();
+
+    public DefaultResponseConfigure(Method method, String path, T testDto, ExecuteQueryToMockInfrastructure executeQuery) {
+        this.method = method;
+        this.path = path;
+        this.executeQuery = executeQuery;
+        this.testDto = testDto;
+    }
+
+    public Method getMethod() {
+        return method;
+    }
+
+    public String getPath() {
+        return path;
+    }
+
+    public T clearDefinedResponses() {
+//        mock.getDynamicRouteStack().clear(method.getHttpMethod(), path);
```

Comment:
Why so many commented code lines?

Reply:
this functionality is disabled temporary. I split the epic to multiple parts but I don't want to remove the old codes. I will re-enable these lines if I can.